_raml/apps/products/examples/get_product_updated_property_response.json

```diff
@@ -143,6 +143,64 @@
             }
           }
         ],
+        "customerSpecificPriceInfo": [
+          {
+            "customerGroup" : "KG1",
+            "customerGroupId": "63E4F781-CDA5-327F-B72B-C0A802027724",
+            "priceInfo": [
+              {
+                "quantity": {
+                  "amount": 1,
+                  "unit":"piece(s)"
+                },
+                "price": {
+                  "taxType": "GROSS",
+                  "formatted": "135.00 €",
```

Comment:
@mhoehn I think that compared to the prices that are listed before in this example, so 54.62 € NET, 27.00 € NET and 36.00 € NET, the prices used for the `customerSpecificPriceInfo` are way to high and therefore unrealistic. I would suggest to lower them and maybe also use NET instead of GROSS?